_posts/en/newsletters/2024-05-24-newsletter.md

```diff
@@ -0,0 +1,381 @@
+  community feedback about mining hardware software features, maintenence, and
+  other questions.
+
+- **Sentrum wallet tracker released:**
+  [Sentrum][gh sentrum] is a watch-only wallet that supports a variety of notification channels.
+
+- **Stack Wallet adds FROST support:**
+  [Stack Wallet v2.0.0][gh stack wallet] adds FROST [threshold][topic threshold
+  signature] multisig support using the Modular FROST rust library.
+
+- **Transaction broadcast tool announced:**
+  [Pushtx][gh pushtx] is a simple rust program that broadcasts transactions directly to the
+  Bitcoin P2P network.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Inquisition 27.0][] is the latest major release of this fork
+  of Bitcoin Core designed for testing soft forks and other major
+  protocol changes on [signet][topic signet].  New in this release is
+  signet enforcement of [OP_CAT][] as specified in [BIN24-1][] and
+  [BIP347][].  Also included is "a new `evalscript` subcommand for
+  `bitcoin-util` that can be used to test script opcode behavior".
+  Support has been dropped for `annexdatacarrier` and pseudo [ephemeral
+  anchors][topic ephemeral anchors] (see Newsletters [#244][news244
+  annex] and [#248][news248 ephemeral]).
+
+- [LND v0.18.0-beta.rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #27101][] introduces support for
+  JSON-RPC 2.0 requests and server responses. Notable changes are that the
+  server always returns HTTP 200 "OK" unless there is an HTTP error or malformed
+  request, it returns either error or result fields but never both, and single
+  and batch requests result in the same error handling behavior. If version 2.0
+  isn't specified in the request body, the legacy JSON-RPC 1.1 protocol is used.
+
+- [Bitcoin Core #30000][] allows multiple transactions with the same
+  `txid` to coexist in `TxOrphanage` by indexing them by `wtxid` instead
+  of `txid`. The orphanage is a memory-limited staging area that Bitcoin Core
+  uses to store transactions which reference the txids of parent
+  transactions that Bitcoin Core doesn't have in its UTXO set or
+  mempool.  If a parent transaction with the txid is received, the child
```

Comment:
```suggestion
  transactions that Bitcoin Core doesn't have in its UTXO set, including its
  mempool.  If a parent transaction with the txid is received, the child
```
I may be wrong here, but I would think a node's UTXO set _includes_ its mempool.